<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeZeroDialog</class>
 <widget class="QDialog" name="EdgeZeroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edge Zero Crossing</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/icons.qrc">
    <normaloff>:/icons/icons/edge_detection.png</normaloff>:/icons/icons/edge_detection.png</iconset>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.200000000000000</double>
   </property>
   <property name="value">
    <double>1.600000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mask size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sigma</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EdgeZeroDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EdgeZeroDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
